posts-api/controllers/post.js

```diff
@@ -3,20 +3,27 @@ const Post = require("../models/post");
 exports.createPost = (req, res, next) => {
 
   const url = req.protocol + "://" + req.get("host");
+  const serviceName = req.body.title.toLowerCase().split(' ').join('-');  
+  console.log("Register service with name "+serviceName);
```

Comment:
serviceName ce biti ruta koju cemo korisniku ponuditi
Znaci ako zeli da gadja swapi servis, nudimo mu rutu
/swapi
Ako neko registruje servis pod imenom Moj Novi Servis
Mi ga cuvamo pod imenom moj-novi-servis, i nudimo posle korisniku rutu
/moj-novi-servis
Znaci korisnik koji zeli da se obrati tom servisu, obraca mu se preko rute
teistool.com/moj-novi-servis

Reply:
BAS TO JE ON HTEO, CARSKI! A sta se desava sa body-em request-a?